app/models/video.py

```diff
@@ -1,4 +1,80 @@
 from app import db
+from flask import abort, make_response
+
 
 class Video(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
\ No newline at end of file
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    release_date = db.Column(db.DateTime)
+    total_inventory = db.Column(db.Integer)
+    customers = db.relationship("Rental", back_populates="video")
```

Comment:
Minor note: consider naming this column rentals to enhance readability

```suggestion
    rentals = db.relationship("Rental", back_populates="video")
```